spec/features/user_deletes_outcome_coverage_from_a_subject_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "User deletes an outcome coverage for a subject" do
+  scenario "coverage has 2 or more outcome coverages" do
+    course = create(:course)
+    outcome_A, outcome_B = create_pair(:outcome)
```

Comment:
I think you can just do `outcomes = create_pair(:outcome)` as you don't ever need to refer to the outcomes indivudually here.

Also, should the outcomes be associated with the course you created in the line above?